build.sbt

```diff
@@ -71,6 +75,25 @@ lazy val browserTestSettings = Seq(
   }
 )
 
+// start a test server before running tests
+// this is required as JavaScript tests run inside a nodejs/browser environment
```

Comment:
sorry to be picky, but this doesn't explain why we need to use a browser env in the first place :)

Reply:
added comment above `browserTestSettings`